src/NHapi.Base/Parser/PipeParser.cs

```diff
@@ -689,22 +695,27 @@ public override string GetVersion(string message)
                     ErrorCode.REQUIRED_FIELD_MISSING);
             }
 
+            if (parserOptions.AllowUnknownVersions)
+            {
+                // TODO: Version.HighestAvailableVersionOrDefault.Version
```

Comment:
Does this TODO need to be addressed in this pull request, or is there a separate issue covering this?

Reply:
It's not required for this PR but something I'd like to pick up at some point.
#400